lhotse/audio.py

```diff
@@ -1215,6 +1215,7 @@ def __init__(
         self.tracks = [base_audio]
         self.offsets = [0]
         self.sampling_rate = sampling_rate
+        self.num_channels = base_audio.shape[0]
```

Comment:
Can you update the documentation of `AudioMixer` to explain the new multi channel behavior? E.g. for K channel base_audio, do you require every mixed track to have K channels as well, or can they also have 1 channel, etc?